parser.py

```diff
@@ -130,43 +127,7 @@ def __init__(self, filePath):
         return
     
     def show(self, data):
-        
-        jamesPrinter = HumanPrinter()
-        jamesPrinter.main(data)
-        '''
-        data = data.replace(', :', ',:')
-        data = data.split(",")
-        x = 0
-        y = 0
-        tabcount = 0
-        tabstring = ""
-        parsestring = ""
-
-        while x < len(data):
-
-            if "[" in data[x]:
-                count = data[x].count("[")
-                y = y + count
-            if "]" in data[x - 1]:
-                count = data[x].count("]")
-                y = y - count
-
-            tabstring = "\n"
-            while tabcount < y:
-                tabstring = tabstring + "  "
-                tabcount = tabcount + 1
-
-            tabcount = 0
-
-            data[x] = tabstring + data[x] + ","
-
-            parsestring = parsestring + data[x]
-
-            x = x + 1
-        
-        print(parsestring)
-        '''
-        return
+        print(transpilers.HumanTextTranspiler.convert_to_string(data))
```

Comment:
This is the main interface anyone outside of lib should think about. Give it the rules, it gives you the string - no state to store or "oops I called these methods out of order and can't tell" problems